www/js/search.js

```diff
@@ -1014,11 +1028,21 @@ module.exports = {
   },
 
   clickShabad(e, ShabadID, LineID, Line, rows, mode = 'click') {
-    /*
     if (window.socket !== undefined) {
-      window.socket.emit('data', { shabadid: ShabadID, highlight: LineID });
+      let shabadIdsplit = [ShabadID];
+      if (typeof ShabadID === 'string') {
+        shabadIdsplit = ShabadID.split('-');
+      }
+
+      window.socket.emit('data', {
+        type: shabadIdsplit.length > 1 ? 'bani' : 'shabad',
```

Comment:
I am little bit confused. How do we differentiate between bani and a ceremony?

Reply:
at the moment ceremony doesn't get the "-" with the bani length and id. not sure if we are going to be supporting ceremony either. might need to test that to see if it breaks anything.